src/app/item-feed/ItemFeed.tsx

```diff
@@ -62,9 +62,10 @@ const Item = memo(function Item({ item, tag }: { item: DimItem; tag: TagValue |
 });
 
 const filteredItemsSelector = createSelector(allItemsSelector, (allItems) =>
-  allItems
-    .filter((i) => i.equipment && i.power > 0 && i.taggable)
-    .sort(acquisitionRecencyComparator)
+  _.sortBy(
+    allItems.filter((i) => i.equipment && i.power > 0 && i.taggable),
+    getItemRecencyKey
```

Comment:
`acquisitionRecencyComparator` wraps this in a `reverseComparator`, so this ends up reversing the item feed, showing oldest items first.